pkg/ccl/backupccl/file_sst_sink.go

```diff
@@ -266,6 +266,7 @@ func (s *fileSSTSink) write(ctx context.Context, resp exportedSpan) (roachpb.Key
 	if err != nil {
 		return nil, err
 	}
+	hasRangeKeys := maxRange != nil
```

Comment:
I am slightly tempted to say this should be an int instead of bool, but I don't have a good reason to say this, and doing so would require extending the return of copyRangeKeys to return the count not just max key.